demo/customer-sdk-demo.html

```diff
@@ -58,7 +58,8 @@
     function onConnect(){
```

Comment:
if you made another method for onConnectWithCustomerAndShare please also rename this one to onConnectWithOrderUuidAndShare for clarity.